src/monodroid/jni/monodroid-glue.cc

```diff
@@ -1813,6 +1829,11 @@ load_assembly (MonoDomain *domain, JNIEnv *env, jstring_wrapper &assembly)
 
 	aname = mono_assembly_name_new (assm_name);
 
+#ifndef ANDROID
+	if (inMemoryAssemblies.has_assemblies () && inMemoryAssemblies.load_assembly_from_memory (domain, aname) != nullptr) {
+		log_info (LOG_DEFAULT, "Dynamically opened assembly %s from memory", mono_assembly_name_get_name (aname));
```

Comment:
Should this really be `log_info()`? Or should this instead be `log_debug()`?  It should probably also use `LOG_ASSEMBLY`?